internal/sqlquery/scraper.go

```diff
@@ -107,3 +109,142 @@ func (s *Scraper) Shutdown(_ context.Context) error {
 	}
 	return nil
 }
+
+func BuildDataSourceString(driver string, dataSource DataSourceConfig) (string, error) {
```

Comment:
This currently returns an error for the `hdb` and `tds` drivers, which are valid and supported.